.github/workflows/zxcron-promote-build-candidate.yaml

```diff
@@ -145,6 +149,74 @@ jobs:
           echo "### Preview Next Build Tag" >> "${GITHUB_STEP_SUMMARY}"
           echo "Next build tag is: ${NEXT_BUILD_TAG}" >> "${GITHUB_STEP_SUMMARY}"
 
+  report-promotion:
+    name: Report Build Promotion
+    runs-on: network-node-linux-medium
+    needs:
+      - determine-build-candidate
+      - promote-build-candidate
+    if: ${{ needs.promote-build-candidate.result == 'success' &&
+            needs.determine-build-candidate.result == 'success' &&
+            needs.determine-build-candidate.outputs.build-candidate-exists == 'true' }}
+    steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@0080882f6c36860b6ba35c610c98ce87d4e2f26f # v2.10.2
+        with:
+          egress-policy: audit
+
+      - name: Report Promoted Build
+        uses: slackapi/slack-github-action@6c661ce58804a1a20f6dc5fbee7f0381b469e001 # v1.25.0
+        env:
+          SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_CITR_OPERATIONS_WEBHOOK }}
+        with:
+          payload: |
+            {
+              "attachments": [
+                {
+                  "color": "#7647cd",
```

Comment:
Is the color purple the right color to use here? When I see purple I think "merge complete" but wanted to see if white or gray made more sense

Reply:
If its a failure, then I would expect to see red or yellow. If its a passing report, I would like to see green or blue. If it is an informational report, then grey or white would be appropriate.

Reply:
I moved the failures to red and the success/promotion to green.